app.rb

```diff
@@ -43,52 +48,57 @@ def add_book
     puts 'Author'
     author = Input.new.input
     book = Book.new(title, author)
+    @books = []
     @books.push(book)
+    write('Data Files/books.json', @books)
     puts 'Book added successfully.'
```

Comment:
When adding a `Book` object, the same scenario when adding a `Student` object happens, the newly added book overwrites the previously added one, and in the end the list only holds the last book added. Kindly fix this issue. Line 51 might have something to do with it.